public/app/features/scopes/dashboards/ScopesDashboardsTree.tsx

```diff
@@ -30,7 +30,7 @@ export function ScopesDashboardsTree({ folders, folderPath, onFolderUpdate }: Sc
       ))}
       {Object.values(folder.suggestedNavigations).map((navigation) => (
         <ScopesNavigationTreeLink
-          key={navigation.id}
+          key={navigation.id + navigation.title}
```

Comment:
Isn't navigation.id unique?

Reply:
It seems to not be the case. At least for the keys. I guess in this case it is conflicting between folders and links.